pkg/acsengine/defaults_test.go

```diff
@@ -430,10 +430,6 @@ func TestNetworkPolicyDefaults(t *testing.T) {
 		t.Fatalf("NetworkPlugin did not have the expected value, got %s, expected %s",
 			properties.OrchestratorProfile.KubernetesConfig.NetworkPlugin, "azure")
 	}
-	if properties.OrchestratorProfile.KubernetesConfig.NetworkPolicy != "" {
```

Comment:
Instead of removing a unit test I think you should update it to the actual expected value that way we don't reduce coverage.